### ISSUE TITLE
Seem to be getting warnings when using ember-cp-validations 

### ISSUE THREAD

**Issue by robbyoconnor** [offirgolan/ember-cp-validations#119]
I'm getting warnings when in my console: 

```
Ember      : 2.5.0-canary+f2684115
Ember Data : 2.5.0-canary+8dc694025e
```
WARNING: Dependent keys containing @each only work one level deep. You cannot use nested forms like todos.@each.owner.name or todos.@each.owner.@each.name. Please create an intermediary computed property.
ember.debug.js:27627

WARNING: Dependent keys containing @each only work one level deep. You cannot use nested forms like todos.@each.owner.name or todos.@each.owner.@each.name. Please create an intermediary computed property.

**Comment by offirgolan** [comment]
Can you add a snippet of your Validations?

On Monday, February 15, 2016, robbyoconnor <notifications@github.com> wrote:

> I'm getting warnings when in my console:
>
> Ember      : 2.5.0-canary+f2684115
> Ember Data : 2.5.0-canary+8dc694025e
>
> WARNING: Dependent keys containing @each <https://github.com/each> only
> work one level deep. You cannot use nested forms like
> todos.@each.owner.name
> <javascript:_e(%7B%7D,'cvml','todos.@each.owner.name');> or todos.@
> each.owner.@each.name
> <javascript:_e(%7B%7D,'cvml','each.owner.@each.name');>. Please create an
> intermediary computed property.
> ember.debug.js:27627
>
> WARNING: Dependent keys containing @each <https://github.com/each> only
> work one level deep. You cannot use nested forms like
> todos.@each.owner.name
> <javascript:_e(%7B%7D,'cvml','todos.@each.owner.name');> or todos.@
> each.owner.@each.name
> <javascript:_e(%7B%7D,'cvml','each.owner.@each.name');>. Please create an
> intermediary computed property.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/offirgolan/ember-cp-validations/issues/119>.
>


-- 
Thanks,
Offir Golan

**Comment by robbyoconnor** [comment]
It was a simple date validator a couple presence validators -- no dependent relationships

**Comment by robbyoconnor** [comment]
It was a simple date validator and a large chunk of it was taken from the dummy app as a reference.

**Comment by offirgolan** [comment]
:+1: thanks for bringing this to my attention. Ill take a deeper look when I have some time

**Comment by offirgolan** [comment]
Fixed with `v2.4.1`

**Comment by robbyoconnor** [comment]
Hi,

I'd like to add you to my professional network on LinkedIn.

- Robert

Confirm that you know Robert: https://www.linkedin.com/e/v2?e=0-ini67ojs-nd&t=ssuw&tracking=eml-guest-invite-cta&ek=invite_guest&sharedKey=2rvcOple&invitationId=6130897019268579328








You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=0-ini67ojs-nd&t=lun&midToken=AQFmrFVhGEd36Q&ek=invite_guest&loid=AQGx4jxtCDN9WQAAAVRVSbl1YSwe7_lJbmq4uXq45BhrHI2B3jP-FuDB6OlYqjGQjHKa5lI52FFV6EcvaJsstH4ArTqIxHtNr_2nNBysifv42VFJAHRFrjuJ-BR5J4m-W8CZZiAiyN7C6qDRZSrXtrb3OyZqXoSwhHnl1kdx5d0V9_MHjuEHLOAearkBaAH2Qa33&eid=0-ini67ojs-nd


This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=0-ini67ojs-nd&a=customerServiceUrl&ek=invite_guest

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.
